src/sql/workbench/browser/modal/modal.ts

```diff
@@ -319,13 +320,21 @@ export abstract class Modal extends Disposable implements IThemable {
 	 * Set focusable elements in the modal dialog
 	 */
 	public setFocusableElements() {
-		this._focusableElements = this._bodyContainer.querySelectorAll('a[href], area[href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), button:not([disabled]), [tabindex="0"]');
+		// try to find focusable element in dialog pane rather than overall container
+		this._focusableElements = this._dialogPaneBody ?
```

Comment:
does it make more sense to just call focus on the modal itself? That way implementers have the chance to define this behavior?